test/cpp/ios/RemoteTestClientCpp/RemoteTestCpp.podspec

```diff
@@ -0,0 +1,93 @@
+Pod::Spec.new do |s|
+  s.name     = "RemoteTestCpp"
+  s.version  = "0.0.1"
+  s.license  = "Apache License, Version 2.0"
+  s.authors  = { 'gRPC contributors' => 'grpc-io@googlegroups.com' }
+  s.homepage = "https://grpc.io/"
+  s.summary = "RemoteTest example"
+  s.source = { :git => 'https://github.com/grpc/grpc.git' }
+
+  s.ios.deployment_target = '7.1'
+  s.osx.deployment_target = '10.9'
+
+  # Run protoc with the C++ and gRPC plugins to generate protocol messages and gRPC clients.
+  s.dependency "!ProtoCompiler-gRPCCppPlugin"
+  s.dependency "Protobuf-C++"
+  s.dependency "gRPC-C++"
+  s.source_files = "src/proto/grpc/testing/*.pb.{h,cc}"
+  s.header_mappings_dir = "RemoteTestCpp"
+  s.requires_arc = false
+
+  repo_root = '../../../..'
+  config = ENV['CONFIG'] || 'opt'
+  bin_dir = "#{repo_root}/bins/#{config}"
+
+  protoc = "#{bin_dir}/protobuf/protoc"
+  well_known_types_dir = "#{repo_root}/third_party/protobuf/src"
+  plugin = "#{bin_dir}/grpc_cpp_plugin"
+  proto_dir = "#{repo_root}/src/proto/grpc/testing"
+
+  s.prepare_command = <<-CMD
+    if [ -f #{protoc} ]; then
+      #{protoc} \
+          --plugin=protoc-gen-grpc=#{plugin} \
+          --cpp_out=. \
+          --grpc_out=. \
+          -I #{repo_root} \
+          -I #{proto_dir} \
+          -I #{well_known_types_dir} \
+          #{proto_dir}/echo.proto
+      #{protoc} \
+          --plugin=protoc-gen-grpc=#{plugin} \
+          --cpp_out=. \
+          --grpc_out=. \
+          -I #{repo_root} \
+          -I #{proto_dir} \
+          -I #{well_known_types_dir} \
+          #{proto_dir}/echo_messages.proto
+      #{protoc} \
+          --plugin=protoc-gen-grpc=#{plugin} \
+          --cpp_out=. \
+          --grpc_out=. \
+          -I #{repo_root} \
+          -I #{proto_dir} \
+          -I #{well_known_types_dir} \
+          #{proto_dir}/simple_messages.proto
+    else
+      # protoc was not found bin_dir, use installed version instead
+      (>&2 echo "\nWARNING: Using installed version of protoc. It might be incompatible with gRPC")
+
+      protoc \
```

Comment:
check if `protoc` exists?

Reply:
We use `protoc` from `bins/$CONFIG/` if available, else we use the system protoc. 
If `protoc` is not present in bins/$CONFIG and also not installed in the system then pod install will fail. We have the same logic in `src/objective-c/tests/RemoteTestClient/RemoteTest.podspec`

Reply:
Yea. I mean, how about provide an error message saying that `protoc` is not found? It's not a big problem so it's up to you.

Reply:
Oh ok makes sense. Fixed.